### PR TITLE
Remove prerender

### PR DESCRIPTION
**Pull request by D-Bits**
Hopefully, this will be the last of the changes.

### CODE REVIEW

File: common/tests/test_helpers.py [hunk 1]
```diff
@@ -9,12 +9,12 @@
 
 
 class CommonHelperTests(TestCase):
-    def test_prerender_urls(self):
+    def test_is_sitemap_urls(self):
         urls = SitemapPages()
         for url in urls:
             self.assertTrue(is_sitemap_url(url), 'Should be able to prerender ' + url)
```
Comment (marlonkeating) [inline common/tests/test_helpers.py:15]: Should update test messages too.

**Review by marlonkeating — CHANGES_REQUESTED** [review]
Can also remove references from

https://github.com/DemocracyLab/CivicTechExchange/blob/ec23e41905d33f92fe07489f50fed2b2717bc2af/democracylab_environment_variables.sh#L114-L115

File: common/tests/test_helpers.py (hunk 1 above)
Comment (D-Bits) [reply common/tests/test_helpers.py]: Just finished removing those from both files.

File: common/components/controllers/AboutProjectController.jsx [hunk 2]
```diff
@@ -90,7 +88,6 @@ class AboutProjectController extends React.PureComponent<{||}, State> {
         <React.Fragment>
           <Helmet>
             <meta
-              name="prerender-status-code"
```
Comment (marlonkeating) [inline common/components/controllers/AboutProjectController.jsx:93]: This whole section should be removed.

**Review by marlonkeating — CHANGES_REQUESTED** [review]

File: common/components/controllers/AboutProjectController.jsx (hunk 2 above)
Comment (D-Bits) [reply common/components/controllers/AboutProjectController.jsx:93]: Just removed it.

**Review by marlonkeating — APPROVED** [review]

**Pull request merged**